lib/game.rb

```diff
@@ -0,0 +1,136 @@
+require_relative 'board'
+require_relative 'player'
+require_relative 'constants'
+
+class Game
+	attr_accessor :board, :turn, :player1, :player2
+	def initialize
+		@board = Board.new
+		@turn = 0
+		@player1 = nil
+		@player2 = nil
+	end
+
+	def add_player(symbol)
+		player_name = gets.chomp
+		return Player.new(player_name, symbol)
+	end
+
+	def move(player)
+		symbol = player.symbol
+		puts "#{player.name} move (print a number from 0 to 8) \n"
+		move_index = gets.chomp
+		if board.matrix[move_index.to_i] == nil && move_index.to_i >= 0 && move_index.to_i < 9 && POSSIBLE_MOVES.any? {|element| element == move_index} 
+			board.change_matrix(player.symbol, move_index.to_i)
+		else
+			puts 'Please make a valid move'
+			move(player)
+		end
+	end	
```

Comment:
Your `move` method does two things, it validates that you get an input between 0 and 8 and that the board is able to write. You can split this in two separate methods for easier read and debugging (among other benefits). A rule of thumb for defining methods/modules or any piece of software is [Do One Thing and Do It Well](https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well).

Split this in two separate methods